scanner/src/test/kotlin/utils/FileListResolverTest.kt

```diff
@@ -57,7 +57,20 @@ class FileListResolverTest : StringSpec({
 
         val fileList = resolver.resolve(ArtifactProvenance(sourceArtifact = RemoteArtifact.EMPTY))
 
-        fileList.toYaml() shouldBe File("src/test/assets/expected-file-list.yml").readText()
+        fileList.ignorePatterns should containExactlyInAnyOrder(
+            "**/.bzr",
+            "**/.git",
+            "**/.hg",
+            "**/.repo",
+            "**/.svn",
+            "**/CVS",
+            "**/CVSROOT"
+        )
+
+        fileList.files should containExactlyInAnyOrder(
```

Comment:
Is the ordering a feature and should we keep on testing it?

Reply:
The type of `files` is `Set<FileEntry>`, so I would say no.